pkg/rancher/migrations.go

```diff
@@ -540,3 +543,69 @@ func insertOrUpdateCondition(d data.Object, desiredCondition summary.Condition)
 
 	return true, nil
 }
+
+func migrateCAPIKubeconfigs(w *wrangler.Context) error {
+	logrus.Info("Running provisioningv2 CAPI kubeconfig secret migration")
+
+	namespaces, err := w.Core.Namespace().List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, ns := range namespaces.Items {
+		provClusters, err := w.Provisioning.Cluster().List(ns.Name, metav1.ListOptions{})
+		if k8serror.IsNotFound(err) || len(provClusters.Items) == 0 {
+			logrus.Tracef("No provisioningv2 clusters in namespace %s", ns.Name)
+			continue
+		} else if err != nil {
+			return err
```

Comment:
Rather than return an error here, maybe we should just `continue`

Reply:
The migration has now been removed.